Vagrantfile

```diff
@@ -10,9 +10,12 @@ Vagrant.configure("2") do |config|
   config.vm.synced_folder "~/.hab/cache/keys", "/hab/cache/keys"
   config.vm.synced_folder "~/.hab/etc", "/hab/etc"
 
-  config.vm.network "forwarded_port", guest: 80, host: 9636
-  config.vm.network "forwarded_port", guest: 9631, host: 9631
-  config.vm.network "forwarded_port", guest: 9636, host: 9636
+  config.vm.network "private_network", ip: "192.168.198.7"
+
+  config.vm.network "forwarded_port", guest: 9631, host: 9631 # Supervisor API
+  config.vm.network "forwarded_port", guest: 9636, host: 9636 # Builder API
+  config.vm.network "forwarded_port", guest: 3000, host: 3000 # Builder UI
```

Comment:
You can change this to 80 as nobody as it's not really tenable to do web dev inside the vagrant vm (file syncing is super slow)

Reply:
Would it be ok to move the UI part into a Docker container instead?